python/paddle/dataset/cifar.py

```diff
@@ -28,7 +28,7 @@
 
 """
 
-import cPickle
+import pickle
```

Comment:
Mayve we should use `six.moves` since `zip` is different with `izip` in Py2

Reply:
Yes, I will change it in next commit